src/main.cpp

```diff
@@ -63,7 +63,9 @@
 #ifdef OSW_FEATURE_WIFI
 #include <services/OswServiceTaskWiFi.h>
 #endif
-
+#ifdef APP_WEATHER
```

Comment:
You must follow the flag rule. `OSW_FEATURE_****`
So `OSW_FEATURE_WEATHER` .

Reply:
in `platform.ini` from user.

Reply:
so is the user that will put it in the platform.ini file?

Reply:
ok, in [97a20b4](https://github.com/Open-Smartwatch/open-smartwatch-os/pull/299/commits/97a20b43374055d97ba6d7a3e96c8c309cd4bf93) it should be as you requested : )

Reply:
Can you upload a video of this code working? I wonder how the UI comes out. Video, please.

Reply:
ok